File: C/Cap03/c03ex02.c
// c03ex02.c

#include <stdio.h>
#include <math.h>

int main(void) {
  double R, A;

  printf("Entre valor do raio: ");
  scanf("%lf", &R);
  while (getchar() != '\n'); 

  A = M_PI * pow(R, 2);

  printf("Resultado = %.2lf\n", A);

  printf("\nTecle <Enter> para encerrar... ");
  while (getchar() != '\n');

  return 0;
}

File: PHP/Cap03/c03ex02.php
<?php
// c02ex01.php

printf("CALCULO DE AREA DE UM CIRCULO\n\n");

printf("Entre valor do raio: ");
$R = floatval(trim(fgets(STDIN)));

$A = M_PI * pow($R, 2);

printf("Resultado = %.2f\n", $A);

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>
